BaseTools/Source/C/Makefiles/ms.common

```diff
@@ -51,6 +51,25 @@ BIN_PATH     = $(BASE_TOOLS_PATH)\Bin\Win64
 LIB_PATH     = $(BASE_TOOLS_PATH)\Lib\Win64
 SYS_BIN_PATH = $(EDK_TOOLS_PATH)\Bin\Win64
 SYS_LIB_PATH = $(EDK_TOOLS_PATH)\Lib\Win64
+CFLAGS = $(CFLAGS) /wd4267 /wd4244 /wd4334
+
+!ELSEIF "$(HOST_ARCH)"=="ARM"
+ARCH_INCLUDE = $(EDK2_PATH)\MdePkg\Include\Arm
+BIN_PATH     = $(BASE_TOOLS_PATH)\Bin\Win32
+LIB_PATH     = $(BASE_TOOLS_PATH)\Lib\Win32
+SYS_BIN_PATH = $(EDK_TOOLS_PATH)\Bin\Win32
+SYS_LIB_PATH = $(EDK_TOOLS_PATH)\Lib\Win32
+
+!ELSEIF "$(HOST_ARCH)"=="AARCH64"
+ARCH_INCLUDE = $(EDK2_PATH)\MdePkg\Include\AArch64
+BIN_PATH     = $(BASE_TOOLS_PATH)\Bin\Win64
+LIB_PATH     = $(BASE_TOOLS_PATH)\Lib\Win64
+SYS_BIN_PATH = $(EDK_TOOLS_PATH)\Bin\Win64
+SYS_LIB_PATH = $(EDK_TOOLS_PATH)\Lib\Win64
+# Note: These are bit-width conversion related warning suppressions we carry to avoid much more
+#       overrides in the C code. The future plan is to replace all of these binary based tools
+#       with python scripts, when it happens in tianocore..

```

Comment:
Also, we're _not_ going to replace the binary tools with Python scripts, because that project has been abandoned in TianoCore. Someone spent a lot of time on it and my understanding is it was wasted since nothing came from it. Let's focus on improving the C code instead.

Reply:
Thanks. I will drop this entire sentence.

Reply:
This sentence was dropped. Thank you for the feedback.